tests/benchmark-middleware-runner.php

```diff
@@ -3,7 +3,7 @@
 use Psr\Http\Message\ServerRequestInterface;
 use React\Http\Io\MiddlewareRunner;
 use React\Http\Io\ServerRequest;
```

Comment:
Old namespace, see also failing job on travis.

```
Fatal error: Uncaught Error: Class 'React\Http\Io\ServerRequest' not found in /home/travis/build/reactphp/http/tests/benchmark-middleware-runner.php:24

Stack trace:

#0 {main}

  thrown in /home/travis/build/reactphp/http/tests/benchmark-middleware-runner.php on line 24
```

Reply:
Thanks for spotting this oversight, fixed :shipit: